lib/telegraf/telegraf.conf.erb

```diff
@@ -616,6 +619,7 @@
   #
   ## Some queries we may want to run less often (such as SHOW GLOBAL VARIABLES)
   interval_slow                   = "30m"
```

Comment:
Actually, I just noticed this in some testing `mysql_users.127_0_0_1.root.connections` so we might need to filter that out.